ambry-router/src/test/java/com.github.ambry.router/ChunkFillTest.java

```diff
@@ -150,11 +149,11 @@ public void testChunkNumAndSizeCalculations() throws Exception {
     MockTime time = new MockTime();
     MockNetworkClientFactory networkClientFactory = new MockNetworkClientFactory(vProps, null, 0, 0, 0, null, time);
     PutOperation op =
-        new PutOperation(routerConfig, routerMetrics, mockClusterMap, responseHandler, new LoggingNotificationSystem(),
-            new InMemAccountService(true, false), putUserMetadata, putChannel, futureResult, null,
-            new RouterCallback(networkClientFactory.getNetworkClient(), new ArrayList<BackgroundDeleteRequest>()), null,
+        PutOperation.forUpload(routerConfig, routerMetrics, mockClusterMap, new LoggingNotificationSystem(),
+            new InMemAccountService(true, false), putUserMetadata, putChannel, new PutBlobOptionsBuilder().build(),
```

Comment:
minor: maybe use `PutBlobOptions.DEFAULT` instead of `new PutBlobOptionsBuilder().build()`?